### PR TITLE
added missing link style

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Missing style makes tests fail

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. review styles

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -105,5 +105,10 @@
                 "https://www.drupal.org/project/menu_link_weight/issues/2995896": "https://www.drupal.org/files/issues/2018-08-29/2995896-menu-admin-per-menu-support-fixes.patch"
             }
         }
+    },
+    "config": {
```
Comment (sherakama) [inline composer.json]: This might cause problems with deploy no? Have the caches cleared and we can remove this now?

Comment (pookmish) [reply composer.json]: I believe its only affects CI tests. but even with this it was still getting inconsistent results when running `composer update`

Comment (sherakama) [reply composer.json]: I think I'd rather the prefer controls be passed in manually than by wiring it up here. Now that some time has passed let me try without this.

Comment (pookmish) [reply composer.json]: yeah i dont know why it was doing that. hours after the last merge, it still wasn't consistent. i almost wonder if a few more commits on `stanford_text_editor` would push it out far enough. :man_shrugging:

**Comment by pookmish** [comment]
double approved :heavy_check_mark: :white_check_mark:

**Pull request merged**